### PR TITLE
CAR-2144 fix opus cmake

### PR DESCRIPTION
**Pull request by lydia-sun**
Changes suggested in CAR-2144 work for carmel, caramilk, caramello, and eastwood but not for Vandenberg.
Vandenberg relied on Airtime specific changes to patch up opus issue which was fixed upstream.
So, solution is to update forked opus with upstream changes.
Edit:
Vandenberg build now after removing cpack from opus

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -258,107 +242,117 @@ set_package_properties(Git
...
                       PROPERTIES SOVERSION
                                  ${OPUS_LIBRARY_VERSION_MAJOR}
                                  VERSION
                                  ${OPUS_LIBRARY_VERSION}
                                  PUBLIC_HEADER
                                  "${Opus_PUBLIC_HEADER}")
-
+# airtime: start                              
+add_library(libopus ALIAS opus)
+# airtime: end
```
Comment (lydia-sun) [inline CMakeLists.txt]: The only non-upstream change for this PR

**Comment by sumitkandpal** [comment]
if caramello etc work with this change. i am good with it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -469,6 +544,10 @@ if(OPUS_INSTALL_PKG_CONFIG_MODULE)
 endif()
 
 if(OPUS_INSTALL_CMAKE_CONFIG_MODULE)
+  # airtime: start
```
Comment (sumitkandpal) [inline CMakeLists.txt:547]: remove cpack

**Review by sumitkandpal — APPROVED** [review]

**Comment by sumitkandpal** [comment]
@artasano  vanderberg builds now so this is good for review. lot of files due to upstream merge but the main changes are
. add_library(libopus ALIAS opus) (some CMakeLists.txt are using libopus instead of opus by mistake so alias was added)
. removing cpack (due to overriding configure values from our cpack)

**Review by artasano — APPROVED** [review]

**Pull request merged**